<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeBulletinsClass</class>
 <widget class="QWidget" name="EmployeeBulletinsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmployeeBulletins</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableBulletins">
      <property name="styleSheet">
       <string notr="true">QTableWidget {
	background: transparent;
}

QHeaderView::section {	
	color: white; 
	background-color: rgba(70, 133, 210, 1);
}

QTableWidget::item:selected {
    background-color: #0e6c96;
	color: #fff;
}</string>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End Date</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="contentTextEdit">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
	background-color: transparent;
	border: 1px solid #000;
}
	</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Content&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
